server.py

```diff
@@ -44,7 +44,8 @@ def signal_handler(signal, frame):
         from docopt import docopt
         args = docopt(__doc__, version='FAF Server')
 
-        logger.info("Using StatsD server: ".format(config.STATSD_SERVER))
+        if config.ENABLE_STATSD:
+            logger.info("Using StatsD server: ".format(config.STATSD_SERVER))
```

Comment:
I may be blind/not have enough knowledge of python, but isn't the parameter never inserted anywhere?

Reply:
Imported config.py as config !